packages/shared/lib/participation/constants.ts

```diff
@@ -44,3 +44,25 @@ export const STAKING_AIRDROP_TOKENS: { [key in StakingAirdrop]: string } = {
  * participation information.
  */
 export const PARTICIPATION_POLL_DURATION = 10 * MILLISECONDS_PER_SECOND
+
+/**
+ * The number of the last completed Assembly staking period.
+ */
+export const LAST_ASSEMBLY_STAKING_PERIOD = 1
+
+/**
+ * The number of the last completed Assembly staking period.
+ * Set to 0 if no current event
+ */
+export const CURRENT_ASSEMBLY_STAKING_PERIOD = 1
+
+/**
+ * The number of the last completed Assembly staking period.
+ */
+export const LAST_SHIMMER_STAKING_PERIOD = 1
+
+/**
+ * The number of the last completed Assembly staking period.
+ * Set to 0 if no current event
+ */
+export const CURRENT_SHIMMER_STAKING_PERIOD = 1
```

Comment:
Hmm should this be set to `0`? I'm not sure the comments for the current staking period constants are correct; perhaps something like "The number of the ongoing Assembly / Shimmer staking period. Set to 0 if no current event."? 

Alternatively this could just be a boolean for if an event is ongoing, then based off of whatever the last staking period is we know what the current period would be (e.g. only if `true` or something).

Reply:
Well it can be left as the previous value if we want. So when we add new event last and current shimmer will be 1. And last assembly will be 1 and current assembly will be 2. Need them both as numbers to compare for logic. I though if we used 0 when we know the event isn't running we can use that somewhere else in the logic.

The new event hasn't been added hence why I have left them in the state they would have been on the previous event.

Reply:
That makes sense. Although the events have technically ended now, so should they still be `1`?

Reply:
Well we haven't released an "update" yet, and for example this code will work with them not being updated between periods when we haven't released an update. But we can change these when we add the event ids.